### DISCUSSION TITLE
Installation Help

### DISCUSSION THREAD

**Discussion by christianreiss** [spali/go-rscp#32] [Q&A] [answered]
Hey!

I desperately need a way to cancel the charging of the wallbox. So I need to send a single parameter to the e3dc via rscp. I go the parameter "ABORT_CHARGING_TYP2" and I need a way to send that to it.

Your application seems like the way to go, but what am I supposed to do with it? No Makefile, no INSTALL...
The referenced "e3dc" binary does not exist. So there is a missing part from "cloning git" to "configuring"

Some help please :)

**Comment by spali** [comment]
Check the releases for pre compiled binaries.
That's the way to get started without messing with code or building it.

The other approach is to use this as a library in go, but then I expect you know how to build or run go code.

**Comment by spali** [comment, marked as answer]
You may also checkout the wiki for examples.

**Comment by christianreiss** [comment]
Hey,

yeah that makes way more sense. Okay, sorry. One more question: The config file should contain:

```
host 127.0.0.1
user myuser
password mypassword
key mykey
```

host - Obvious the fqdn/ip of the e3dc system, check.
user and password? what key?
From a paralell project: https://github.com/git-kick/ioBroker.e3dc-rscp
username and password are e3dc login for the web portal.

Could you please clarify? Thanks :)

**Comment by christianreiss** [comment]
From the wiki:

"create a config file with your details (replace the <> values with your details). The aes key can be changed in the Web-GUI of the e3dc under "Personalisieren" -> "Benutzerprofil"."

That explains the key. But user/ pass remain open?

**Reply by spali** [reply]
User/password is the login you use to login in the local web gui or the online e3dc web portal.

**Comment by christianreiss** [comment]
There is a local web portal? 80/443 is closed.
I am logged in on the e3dc (local console) using the e3dc cloud credentials.

I can set two rscp passwords: One for farming, one for local.

As you wrote here
https://github.com/spali/go-rscp/discussions/16#discussioncomment-2680288
Implementation is only for local accounts?

So...
Username and Password from the cloud login (https://s10.e3dc.com/)  and key is the rscp password for local?

Thank you for all your help.

**Reply by spali** [reply]
Mine has no local console but local web gui, which has the same login as the online portal. But can't remember if it has to be the same. But you can just try it out. Shouldn't harm anything.

**Comment by christianreiss** [comment]
Issue resolved; able to log in and do stuff :)
Thanks!
